docs/components/concepts/process-instance-migration.md

```diff
@@ -177,6 +177,7 @@ The following limitations exist that may be supported in future versions:
 - The following scenarios cannot be migrated:
   - A process instance with an incident
   - A process instance that is started from a call activity, i.e. a child process instance
+  - A process instance with a service task when a boundary event is active
```

Comment:
:x: The limitation is not completely correct.

```suggestion
  - A process instance with an active service task that has a boundary event
```

Reply:
@abbasadel Following limitations should also be added:

```md
- A target process definition with a service task that has a boundary event
- A process instance that contains event subprocess
- A target process definition that contains event subprocess
```